src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/deployment/global.yml

```
@@ -1118,12 +1118,29 @@ Resources:
                     ],
                     "Default": "EnableCrossAccountAccess"
                 },
-                "EnableCrossAccountAccess": {
-                    "Type": "Task",
-                    "Resource": "${EnableCrossAccountAccess.Arn}",
-                    "Next": "UpdateDeploymentPipelines",
-                    "TimeoutSeconds": 900
-                },
+                "EnableCrossAccountAccessMap": {
+                      "Type": "Map",
+                      "Next": "UpdateDeploymentPipelines",
+                      "Iterator": {
+                        "StartAt": "EnableCrossAccountAccess",
+                        "States": {
+                          "EnableCrossAccountAccess": {
+                            "Type": "Task",
+                            "Resource": "${EnableCrossAccountAccess.Arn}",
+                            "TimeoutSeconds": 900,
+                            "End": true
+                          }
+                        }
+                      },
+                      "ItemsPath": "$.account_ids",
+                      "Parameters": {
+                        "deployment_account_region.$": "$.deployment_account_region",
+                        "deployment_account_id.$": "$.deployment_account_id",
+                        "account_id.$": "$$.Map.Item.Value",
+                        "regions.$": "$.regions"
+                      },
+                      "ResultPath": null
+                    }
```

I think there is missing comma here:
```suggestion
                    },
```

---

Oddly enough, the next line indents on a slightly different line it seems. Can you check this?